### PR TITLE
accel/decel button metric increments

### PR DESCRIPTION
**Pull request by speedking456**
Freedom units have weird multiples compared to metric. Modified so that it increments by 10 km/h on long press and 1 km/h on short press, rather than 8 and 1.6 km/h. This mimics stock hyundai behaviour on metric vehicles.

### CODE REVIEW

File: selfdrive/controls/lib/drive_helpers.py [hunk 1]
```diff
@@ -1,16 +1,25 @@
 import math
 from cereal import car
 from common.numpy_fast import clip, interp
+from common.params import Params
 from common.realtime import DT_MDL
 from selfdrive.config import Conversions as CV
 from selfdrive.modeld.constants import T_IDXS
 
+IS_METRIC = Params().get_bool("IsMetric")
```
Comment (adeebshihadeh) [inline selfdrive/controls/lib/drive_helpers.py:9]: this needs to update as the toggle is flipped

Comment (speedking456) [reply selfdrive/controls/lib/drive_helpers.py:9]: Gonna need some help. I'm not sure how I would go about doing that

**Comment by pd0wm** [comment]
You're sure hyundai uses 10 km/h increments? Think Honda uses 5 km/h.

**Comment by KyleGreenlaw** [comment]
Can confirm Hyundai stock system uses +/- 10km/h on long press.

**Comment by KyleGreenlaw** [comment]
Also min set speed on Hyundai stock is 30km/h, not 40.

**Comment by pd0wm** [comment]
ISO 15622:2018 suggests a min set speed of 4.4 m/s.

**Comment by spektor56** [comment]
> You're sure hyundai uses 10 km/h increments? Think Honda uses 5 km/h.

Yea Honda increments by 5 and for Canadians makes more sense.  Most people drive above speed limits and you can go 15 km/h over without getting demerit points.  65 on a 50, 75 on a 60 etc.

**Comment by pd0wm** [comment]
What change broke things? The short/long press, or the metric one?

**Pull request merged**